services/graphql-server/src/graphql/resolvers/website/section.js

```diff
@@ -60,6 +60,21 @@ module.exports = {
       return sections.reverse();
     },
 
+    /**
+     * Returns the website sections directly related to this section. This is primarily used for the
+     * Leaders Program to denote a contextual relationship between two website sections.
+     */
+    relatedSectionIds: async ({ relatedSections }) => relatedSections || [],
```

Comment:
Convert ID resolvers to act like the following:
`relatedSectionIds: (doc) => getAsArray(doc, 'relatedSections'),`